ui/admin/app/components/host-catalog-type-badge/index.hbs

```diff
@@ -1,15 +1,15 @@
 {{#if @model.isStatic}}
-  <Rose::Badge>
-    {{t (concat 'resources.host-catalog.types.' @model.compositeType)}}
-  </Rose::Badge>
+  <Hds::Badge
```

Comment:
No blocker:
Code block from line 2 to 4 is the same as 6 to 8.

To avoid repeat code, what do you think of refactoring the conditional to use two conditions? since we already have [ember-trut-helpers](https://github.com/jmurphyau/ember-truth-helpers#readme) something like:

```html
{{#if (or @model.isStatic @model.isUnknown)}}
  ...code
{{else}}
  ...code
{{/if}}
```

Reply:
I believe we discussed not using helper methods like or or and in templates to keep it more readable. In my case, I moved the logic into the controller. We could do the same here.

Reply:
I think a simple `or` or `and` are fine, I do think if there's negation involved with the `or` or `and` (it gets nested) then it might be prone to logic errors or hard to read.

I'd be fine with this `or` personally but I'd like to hear when we should use `ember-truth-helpers` if not

Reply:
Good point! I moved this conditional check to the component js file

Reply:
I prefer to keep the logic out of the component/template and keep it simple. And this would also work out perfectly if we add more host catalog types.

Reply:
Moving the logic to the controller works for me 😄  That's a valid solution, and the de-facto solution if you are not using ember-truth-helpers.

Reply:
Taking a step back from the code, the logic seems to be something like "if this is a known plugin, display the badge with an icon otherwise show a text badge".  If that's the case, perhaps the model itself should have an `isKnownPlugin` getter.  You could then flip this else/if order and you won't need a component JS file.

I'd also reexamine the existing getters related to types on host catalogs, sets, and hosts in general.  Do they all make sense?

Reply:
However, there won't necessarily be a 1-to-1 mapping between the plugin name (even if known) and Flight icons.  Perhaps this could be represented using a similar mapping strategy as what was used for session badges?  i.e. keep a hash of the mappings in the component.

Reply:
that's a valid point! I'll investigate more and update the pr